ferrocene/doc/user-manual/src/targets/index.rst

```diff
@@ -215,8 +243,42 @@ should not be used in production.
      - \-
 
 
-If your project needs support for one of these targets, please reach out to the
-Ferrocene support team.
+Experimental Targets
+--------------------
+
+Experimental targets are provided as a preview and without other assurances.
+Experimental targets are not suitable for production without extensive user testing.
+
+Limited support available. Known problems are not tracked.
+
+The Ferrocene test suite may have been executed at some point.
+
+Experimental targets may be removed for any reason without warning.
```

Comment:
Experimental targets may be removed for any reason without warning or deprecation period.